src/api/types.ts

```diff
@@ -79,3 +80,22 @@ export type RawTransactionData = {
 };
 
 export type SwapAPIParams = GetQuoteAPIParams | GetSwapAPIParams;
+
+/**
+ * Note: exported as public Type
+ */
+export type HydrateChargeParams = {
+  sender: Address; // The address of the wallet paying
```

Comment:
in OnchainKit when we refer to the wallet in charge, we call it `address: Address`.

As it simplify the conversation as this is the account address in charge of this operation.

Using the word `sender` put the conversation as the API is the boss, and it says "who is the sender?" but the boss is the developer that using their own wallet, or customer wallet.

So if you say to the developer give me the `sender`, I honestly think "sender who? who is sending what? do I have an amazon package to deliver?" it just doesn't make sense.

So when we tell the story, the developer is at the center of our wording, not the backend.

Reply:
I'm fine with having the developer API refer to it as address, and keep sender as an API/Contract concept. Updated.